src/game/client/components/menus_ingame.cpp

```diff
@@ -697,44 +699,46 @@ void CMenus::RenderServerControl(CUIRect MainView)
 	Extended.HSplitTop(20.0f, &Note, &Extended);
 	Extended.HSplitTop(20.0f, &Bottom, &Extended);
 	{
-		Bottom.VSplitRight(120.0f, &Bottom, &Button);
-
-		// render kick reason
-		CUIRect Reason, ClearButton, Label;
-		Bottom.VSplitRight(40.0f, &Bottom, 0);
-		Bottom.VSplitRight(160.0f, &Bottom, &Reason);
-		Reason.VSplitRight(Reason.h, &Reason, &ClearButton);
-		const char *pLabel = Localize("Reason:");
-		float w = TextRender()->TextWidth(0, Reason.h*ms_FontmodHeight*0.8f, pLabel, -1);
-		Reason.VSplitLeft(w + 10.0f, &Label, &Reason);
-		Label.y += 2.0f;
-		UI()->DoLabel(&Label, pLabel, Reason.h*ms_FontmodHeight*0.8f, CUI::ALIGN_LEFT);
-		static float s_Offset = 0.0f;
-		DoEditBox(&m_aCallvoteReason, &Reason, m_aCallvoteReason, sizeof(m_aCallvoteReason), Reason.h*ms_FontmodHeight*0.8f, &s_Offset, false, CUI::CORNER_L);
-
-		// clear button
-		{
-			static CButtonContainer s_ClearButton;
-			float Fade = ButtonFade(&s_ClearButton, 0.6f);
-			RenderTools()->DrawUIRect(&ClearButton, vec4(1.0f, 1.0f, 1.0f, 0.33f+(Fade/0.6f)*0.165f), CUI::CORNER_R, 3.0f);
-			Label = ClearButton;
+		if(Authed || m_pClient->m_aClients[m_pClient->m_LocalClientID].m_Team != TEAM_SPECTATORS){
+			Bottom.VSplitRight(120.0f, &Bottom, &Button);
+			
+			// render kick reason
+			CUIRect Reason, ClearButton, Label;
+			Bottom.VSplitRight(40.0f, &Bottom, 0);
+			Bottom.VSplitRight(160.0f, &Bottom, &Reason);
+			Reason.VSplitRight(Reason.h, &Reason, &ClearButton);
+			const char *pLabel = Localize("Reason:");
+			float w = TextRender()->TextWidth(0, Reason.h*ms_FontmodHeight*0.8f, pLabel, -1);
+			Reason.VSplitLeft(w + 10.0f, &Label, &Reason);
 			Label.y += 2.0f;
-			UI()->DoLabel(&Label, "x", Label.h*ms_FontmodHeight*0.8f, CUI::ALIGN_CENTER);
-			if(UI()->DoButtonLogic(s_ClearButton.GetID(), "x", 0, &ClearButton))
-				m_aCallvoteReason[0] = 0;
-		}
+			UI()->DoLabel(&Label, pLabel, Reason.h*ms_FontmodHeight*0.8f, CUI::ALIGN_LEFT);
+			static float s_Offset = 0.0f;
+			DoEditBox(&m_aCallvoteReason, &Reason, m_aCallvoteReason, sizeof(m_aCallvoteReason), Reason.h*ms_FontmodHeight*0.8f, &s_Offset, false, CUI::CORNER_L);
 
+			// clear button
+			{
+				static CButtonContainer s_ClearButton;
+				float Fade = ButtonFade(&s_ClearButton, 0.6f);
+				RenderTools()->DrawUIRect(&ClearButton, vec4(1.0f, 1.0f, 1.0f, 0.33f+(Fade/0.6f)*0.165f), CUI::CORNER_R, 3.0f);
+				Label = ClearButton;
+				Label.y += 2.0f;
+				UI()->DoLabel(&Label, "x", Label.h*ms_FontmodHeight*0.8f, CUI::ALIGN_CENTER);
+				if(UI()->DoButtonLogic(s_ClearButton.GetID(), "x", 0, &ClearButton))
+					m_aCallvoteReason[0] = 0;
+			}
+		}
+ 
 		if(pNotification == 0)
 		{
 			// call vote
 			static CButtonContainer s_CallVoteButton;
 			if(DoButton_Menu(&s_CallVoteButton, Localize("Call vote"), 0, &Button) || doCallVote)
 				HandleCallvote(s_ControlPage, false);
 		}
-		else
+		else if (!Authed)
```

Comment:
If the notice shouldn't be shown when `!Authed`, then the Call Vote button should be available

Reply:
The call vote button was not there before my commit even when authed so I did not change that. If an admin wants to vote in spec, it has to be a force vote (same as in 0.6). Therefore the notification doesnt concern the admin.

Reply:
Oh I see, my bad then.

Still, that looks like a significant shortcoming of the UI that should be addressed at some point.